components/raftstore-v2/src/worker/tablet_flush.rs

```diff
@@ -63,12 +63,13 @@ impl<EK: KvEngine, ER: RaftEngine> Runner<EK, ER> {
             .get(region_id)
             .map(|mut cache| cache.latest().cloned()) else {return};
         let now = Instant::now();
-        tablet.flush_cfs(DATA_CFS, true).unwrap();
+        // sync flush for leader to let the flush happend before later checkpoint.
+        tablet.flush_cfs(DATA_CFS, is_leader).unwrap();
         let elapsed = now.saturating_elapsed();
         // to be removed after when it's stable
         info!(
```

Comment:
how about use metrics?

Reply:
It's temporarily used here to debug and diagnose.